src/mock/user/interface.ts

```diff
@@ -0,0 +1,3 @@
+import { FileBox, UrlLink, MiniProgram } from 'wechaty'
+import { ContactMock } from './contact-mock'
+export type Attachment = FileBox | UrlLink | MiniProgram | ContactMock
```

Comment:
Please rename this file to `types.ts` to follow the naming style from other Wechaty projects.